### PR TITLE
CAMEL-19005: Upgrade to 6.3.1

### PR DESCRIPTION
**Pull request by Croway**
# Description

# Target

- [ ] I checked that the commit is targeting the correct branch (note that Camel 3 uses `camel-3.x`, whereas Camel 4 uses the `main` branch)

# Tracking
- [ ] If this is a large change, bug fix, or code improvement, I checked there is a [JIRA issue](https://issues.apache.org/jira/browse/CAMEL) filed for the change (usually before you start working on it).

# Apache Camel coding standards and style

- [ ] I checked that each commit in the pull request has a meaningful subject line and body.

- [ ] I formatted the code using `mvn -Pformat,fastinstall install && mvn -Psourcecheck`

Verified itest

image

### CODE REVIEW

File: components/camel-pubnub/src/main/java/org/apache/camel/component/pubnub/PubNubConsumer.java [hunk 1]
```diff
@@ -131,6 +138,41 @@ public void presence(PubNub pubnub, PNPresenceEventResult presence) {
             }
         }
 
+        @Override
+        public void signal(@NotNull PubNub pubnub, @NotNull PNSignalResult pnSignalResult) {
+            System.out.println("A");
```
Comment (oscerd) [inline components/camel-pubnub/src/main/java/org/apache/camel/component/pubnub/PubNubConsumer.java:143]: Are you sure about this println?

**Review by oscerd — CHANGES_REQUESTED** [review]
One minor finding. This will help a lot with some CVEs

File: components/camel-pubnub/src/main/java/org/apache/camel/component/pubnub/PubNubConsumer.java (hunk 1 above)
Comment (Croway) [reply components/camel-pubnub/src/main/java/org/apache/camel/component/pubnub/PubNubConsumer.java]: ops!, not really :) anyway tests that are present in pubnub do not use these features

**Review by davsclaus — APPROVED** [review]

**Comment by Croway** [comment]
failures in PR tests are not related to the changes.

**Comment by davsclaus** [comment]
Okay you are welcome to merge

**Pull request merged**